chia/timelord/timelord_launcher.py

```diff
@@ -18,22 +19,25 @@
 from chia.util.network import resolve
 from chia.util.setproctitle import setproctitle
 
-active_processes: List = []
-stopped = False
-
 log = logging.getLogger(__name__)
 
 
-async def kill_processes(lock: asyncio.Lock):
-    global stopped
-    global active_processes
-    async with lock:
-        stopped = True
-        for process in active_processes:
+@dataclass
+class VDFClientProcessMgr:
+    lock: asyncio.Lock
+    stopped: bool
```

Comment:
And associated call-site simplifications.

```suggestion
    lock: asyncio.Lock = field(default_factory=asyncio.Lock)
    stopped: bool = False
```

Reply:
Added